src/execute.ts

```diff
@@ -34,7 +34,7 @@ import {settings} from './settings'
  *
  * @example For CJS dev.js
  * ```
- * #!/usr/bin/env ts-node
+ * #!/usr/bin/env -S node --loader ts-node/esm --no-warnings=ExperimentalWarning
```

Comment:
I'll defer to you, but all of this ESM stuff is new to me... but I would've expected the CJS to remain the same?